megamek/src/megamek/client/ui/swing/lobby/LobbyMekPopup.java

```diff
@@ -372,6 +374,23 @@ private static JMenu squadronMenu(ClientGUI cg, boolean enabled, ActionListener
         return menu;
     }
     
+    /**
+     * Returns the "Priority Target" submenu, allowing to assign units as a strategic
+     * target to a local Princess bot.
+     */
+    private static JMenu prioTargetMenu(ClientGUI cg, boolean enabled, ActionListener listener,
+            Collection<Entity> entities) {
+
+        JMenu menu = new JMenu("Set Prio Target for");

```

Comment:
I'm sure that we have enough bits that we can spell "Priority" fully.

Reply:
OK, done

Reply:
OK, done